.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Delete this folder, Add your project in cards.json and README

Reply:
which folder ?